Example/ViewController.swift

```diff
@@ -486,7 +494,22 @@ class ViewController: UIViewController {
     func requestIdleCamera() {
         navigationMapView.navigationCamera.stop()
     }
+    
+    func requestAutoDirections() {
+        profileIdentifier = .automobileAvoidingTraffic
+        requestRoute()
+    }
+    
+    func requestCyclingDirections() {
+        profileIdentifier = .cycling
+        requestRoute()
```

Comment:
Looks like you don’t have to request a route first before choosing one of these options, which is nice. But there’s no way to see which one you’re requesting before you long-tap. I wonder if that will be confusing. How about changing the title of the “Start Navigation” button to “Start Driving” etc. based on the current routing profile?

Reply:
This is a good idea -- I'll implement this change.